metrics/storage/fio-k8s/README.md

```diff
@@ -0,0 +1,33 @@
+# FIO test in K8s.
```

Comment:
Document that this requires a k8s cluster ready with kata as runtimeclass. The runtimeclass name should be `kata` or document how to edit to fit into the name of the runtimeclass.